spec/validators/outgoings_assessment/answers_validator_spec.rb

```diff
@@ -48,17 +49,14 @@
           instance_double(Payment, complete?: false)
```

Comment:
This is a dulicate here:
` allow(record).to receive_message_chain(:outgoings_payments, :council_tax) {
          instance_double(Payment, complete?: false)
        }`